packages/nextjs/src/utils/metadata.ts

```diff
@@ -0,0 +1,23 @@
+import { SDK_VERSION } from '@sentry/core';
+import { Options, SdkInfo } from '@sentry/types';
+
+const PACKAGE_NAME_PREFIX = 'npm:@sentry/';
+
+/**
+ * A builder for the SDK metadata in the options for the SDK initialization.
+ * @param options sdk options object that gets mutated
+ * @param names list of package names
+ */
+export function buildMetadata(options: Options, names: string[]): void {
+  options._metadata = options._metadata || {};
+  options._metadata.sdk =
+    options._metadata.sdk ||
+    ({
+      name: 'sentry.javascript.nextjs',
+      packages: names.map(name => ({
+        name: `${PACKAGE_NAME_PREFIX}/${name}`,
```

Comment:
`/` is already in the prefix.

```suggestion
        name: `${PACKAGE_NAME_PREFIX}${name}`,
```

Reply:
added tests!